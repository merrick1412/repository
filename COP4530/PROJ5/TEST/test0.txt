test acm, cah
test brd
test olh
