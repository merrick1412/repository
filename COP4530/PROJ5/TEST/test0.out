test act, car
test bad
test och
